app/views/practices/show/show.html.erb

```diff
@@ -103,8 +103,8 @@
 <% end %>
 
 <!-- Diffusion Tracker -->
-<% diffusion_histories = @practice.diffusion_histories.includes(:va_facility).joins(:va_facility).order('va_facilities.street_address_state, va_facilities.official_station_name') %>
-<% if diffusion_histories.any? && ENV['GOOGLE_API_KEY'] %>
+<% diffusion_histories = @practice.diffusion_histories %>
+<% if diffusion_histories.any? && diffusion_histories.exclude_clinical_resource_hubs.any? && ENV['GOOGLE_API_KEY'] %>
```

Comment:
do we need `diffusion_histories.exclude_clinical_resource_hubs.any? &&`? since we want to display this section if there are diffusion histories of any sort

Reply:
I'm not sure if we do. Are you asking if we want to display the adoption accordions with no map? If that's the case, wouldn't we need a new header, or would we keep it:

**Diffusion tracker**
**Does not include Clinical Resource Hubs (CRH)**

Reply:
This may need to be a design question, but I was under the assumption we wouldn't show the map/accordions if a practice ONLY had CRH adoptions. I'll ask Becca/Tomi/Athena